Resources/Prototypes/Corvax/Recipes/Reactions/drinks.yml

```
@@ -0,0 +1,159 @@
+      amount: 1
+    Water:
+      amount: 1
+    Sugar:
+      amount: 1
+  products:
+    OldFashioned: 10
+
+- type: reaction
+  id: BadTouch
+  reactants:
+    Vodka: 
```

yamllint

[error] trailing spaces

(trailing-spaces)